benchmark/echo/README.md

```diff
@@ -0,0 +1,27 @@
+# Echo Server Benchmark
+
+A simple echo server benchmark. A client sequentially sends requests to the
+server and waits for the response. After getting the response, the client 
+retries. We measured the time it takes to finish this procedure for 10,000
+requests. We tested PGo compiled Go output versus a handwritten echo server
+in Go. The handwritten server uses raw TCP connections. The send and
+received messages are the same in both systems. So the  serailization/deserialization 
+cost for both systems should be equal.
+The tests are done on an M1 Macbook Air.
+
+```
+$ go run cmd/handwritten/main.go
+2021/11/08 17:46:07 duration: 1.281992334s
+```
+
+```
+# using relaxed mailboxes
+$ go run cmd/pgo/main.go
```

Comment:
Maybe add a quick note that the difference between relaxed / full mailboxes is to comment / uncomment one line in the setup code, otherwise it looks like someone accidentally pasted the same command-line twice.